hotel-api-services/src/main/java/com/fponce/hotelapp/hotel/service/HotelService.java

```diff
@@ -0,0 +1,10 @@
+package com.fponce.hotelapp.hotel.service;
```

Comment:
Take care about the exception and some documentation wouldn't be wrong. Remember that the interface is the public contract from the app.

Reply:
I added proper exception handling, and also proper documentation. I followed https://dev.liferay.com/participate/javadoc-guidelines
https://zeroturnaround.com/rebellabs/reasons-tips-and-tricks-for-better-java-documentation/
hope it is fine.